File: erezervisi_desktop/erezervisi_desktop/lib/models/responses/reservation/reservation_get_short_dto.dart
class ReservationGetShortDto {
  late num id;
  late String guest;
  late String accommodationUnit;
  String? guestImage;
  late DateTime from;
  late DateTime to;
  late num totalPrice;

  ReservationGetShortDto({
    required this.id,
    required this.accommodationUnit,
    required this.guest,
    this.guestImage,
    required this.from,
    required this.to,
    required this.totalPrice,
  });

  factory ReservationGetShortDto.fromJson(Map<String, dynamic> json) {
    return ReservationGetShortDto(
      id: json['id'] as num,
      guest: json['guest'],
      accommodationUnit: json['accommodationUnit'],
      guestImage: json['guestImage'],
      from: DateTime.parse(json['from']),
      to: DateTime.parse(json['to']),
      totalPrice: json['totalPrice'] as num,
    );
  }
}

File: erezervisi_api/eRezervisi/eRezervisi.Common.Dtos/Reservation/ReservationGetShortDto.cs
namespace eRezervisi.Common.Dtos.Reservation
{
    public class ReservationGetShortDto
    {
        public long Id { get; set; }
        public string Guest { get; set; } = null!;
        public DateTime From { get; set; }
        public DateTime To { get; set; }
        public double TotalPrice { get; set; }
        public string AccommodationUnit { get; set; } = null!;
    }
}
